packages/sales-api-service/src/services/exceptions/exceptions.service.js

```diff
@@ -1,6 +1,9 @@
 import { persist, StagingException, PoclStagingException } from '@defra-fish/dynamics-lib'
 import db from 'debug'
 import { getGlobalOptionSetValue } from '../reference-data.service.js'
+
+// @IWTF-2174: Remove after testing
+import { PoclDataValidationError } from './temp/index.js'
```

Comment:
is this to be removed once Naimulla has tested? Or can it be removed now?

Reply:
This will be removed before it is merged into the develop branch, but will need to be left in for now so that I can test the next part of the work and the process as a whole before it's merged in.